tests/Backend/FileWorkspace/ComponentsWorkspacesTest.php

```diff
@@ -92,8 +92,10 @@ public function testWorkspaceCreate(): void
         $this->assertCount(2, $componentWorkspacesIds);
         $this->assertEquals($workspacesIds, $componentWorkspacesIds);
 
+        /** @var array $connection */
```

Comment:
todle nevim, jen jsem to anotoval. nenasel jsem, kde by se delal arrayAccess. podezrivam toto https://github.com/keboola/storage-api-php-client/blob/c3ee94bb2fc3e995adc14f1232eb6852983af9df/src/Keboola/StorageApi/Workspaces.php#L147

Reply:
Bude to dělat ten assert array has key! Ten má tenhle typehint. A ta metoda na vytvoření WS nemá typehint. Takže se to asi "nachytří" z toho assertu.